### PR TITLE
Migrate [level 2] packages to Emotion

### PR DESCRIPTION
**Pull request by simonsinclair**
Relates to https://github.com/bbc/psammead/issues/3763.

**Overall change:** 

- psammead-timestamp
- psammead-sitewide-links
- psammead-brand
- psammead-byline
- psammead-grid
- psammead-story-promo
- psammead-social-embed 
- psammead-paragraph
- psammead-headings
- psammead-useful-links
- psammead-script-link
- psammead-image-placeholder
- psammead-section-label
- psammead-bulleted-list
- psammead-heading-index
- psammead-consent-banner
- psammead-copyright

**Code changes:**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-brand/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 7.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Migrate to Emotion. |
```
Comment (simonsinclair) [inline packages/components/psammead-brand/CHANGELOG.md]:
```suggestion
| 7.0.0 | [PR#3891](https://github.com/bbc/psammead/pull/3891) Migrate to Emotion. |
```

**Review by ryanmccombe — COMMENTED** [review]
Still need to add PR numbers to changelog

File: packages/components/psammead-byline/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 | Version       | Description                                                                                               |
 | ------------- | --------------------------------------------------------------------------------------------------------- |
+| 3.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Migrate to Emotion. |
```
Comment (simonsinclair) [inline packages/components/psammead-byline/CHANGELOG.md]:
```suggestion
| 3.0.0 | [PR#3891](https://github.com/bbc/psammead/pull/3891) Migrate to Emotion. |
```

File: packages/components/psammead-grid/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Migrate to Emotion. |
```
Comment (simonsinclair) [inline packages/components/psammead-grid/CHANGELOG.md]:
```suggestion
| 3.0.0 | [PR#3891](https://github.com/bbc/psammead/pull/3891) Migrate to Emotion. |
```

File: packages/components/psammead-sitewide-links/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Migrate to Emotion. |
```
Comment (simonsinclair) [inline packages/components/psammead-sitewide-links/CHANGELOG.md]:
```suggestion
| 6.0.0 | [PR#3891](https://github.com/bbc/psammead/pull/3891) Migrate to Emotion. |
```

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 8.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Talos - Bump Dependencies - @bbc/psammead-styles |
```
Comment (simonsinclair) [inline packages/components/psammead-story-promo/CHANGELOG.md]:
```suggestion
| 8.0.0 | [PR#3891](https://github.com/bbc/psammead/pull/3891) Talos - Bump Dependencies - @bbc/psammead-styles |
```

File: packages/components/psammead-story-promo/src/TextStyles.jsx [hunk 6]
```diff
@@ -19,9 +19,7 @@ const fullWidthColumnsMaxScaleable = `100%`;
 
 const halfWidthColumnsMaxScaleable = `50%`;
 
-const paddingDir = ({ dir }) => `padding-${dir === 'rtl' ? 'left' : 'right'}`;
-
-const TextGridColumnsTopStory = css`
+const TextGridColumnsTopStory = `
```
Comment (ryanmccombe) [inline packages/components/psammead-story-promo/src/TextStyles.jsx:22]: Did we do a visual comparison on storybook between this branch and the storybook on github?  Lot of changes here 👍

File: packages/components/psammead-timestamp/CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Migrate to Emotion. |
```
Comment (simonsinclair) [inline packages/components/psammead-timestamp/CHANGELOG.md]:
```suggestion
| 4.0.0 | [PR#3891](https://github.com/bbc/psammead/pull/3891) Migrate to Emotion. |
```

**Review by ryanmccombe — APPROVED** [review]

**Review by RichardPK — APPROVED** [review]
Version bumps on changelogs and then good!

**Pull request merged**